docs/modules/invoices-and-payments.md

```
@@ -97,4 +100,4 @@ The owner dashboard's Revenue This Week KPI uses the same weekly Payment-ledger
 
 ## Last verified date
 
-2026-08-10
+2026-08-20
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)